be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/LaoHandler.scala

```diff
@@ -48,16 +50,24 @@ case object LaoHandler extends MessageHandler {
           case DbActorAck() => {
             val ask: Future[GraphMessage] = (dbActor ? DbActor.Write(channel, message)).map {
               case DbActorWriteAck() => {
-                val socialChannel: Channel = Channel(channel + Channel.SOCIAL_MEDIA_POSTS_PREFIX)
+                val socialChannel: Channel = Channel(channel + Channel.SOCIAL_MEDIA_CHIRPS_PREFIX)
```

Comment:
the nesting in this function has become crazy and the readability is somewhat bad. 
It seems that we're creating a ton of nested futures and awaiting for them.
How about moving some of the logic to the DbActor so that multiple channels can be created with a single command ? 

something akin to (pseudocode)
```
val ask: Future[GraphMessage] = (dbActor ? DbActor.CreateChannels([
   (laoChan, ObjectType.LAO),
   (socChan, ObjectType.CHIRP),
   (reacChan, ObjectType.REACTION),
]))
```

Reply:
this is a good reason for refactoring, adding this